.github/ISSUE_TEMPLATE/general-report--issue-with-articles.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 'Bug Report: Issues with \news or guide articles or documentation'
+about: Report issue with a specific article, like broken links, typos, missing parts, etc
+title: ''
+labels: 'status: needs help for triage, type: bug'
+assignees: ''
+
+---
+
+<!-- 
+NOTE: If you want to become an author of freecodecamp, you can find everything here: https://www.freecodecamp.org/news/developer-news-style-guide/
```

Comment:
```suggestion
NOTE: If you want to become an author on freeCodeCamp, you can find everything here: https://www.freecodecamp.org/news/developer-news-style-guide/
```
Nitpicks. 'on' makes is clear that they're producing content on the platform, not producing the platform itself.